<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>swathProfileDialogBase</class>
 <widget class="QDialog" name="swathProfileDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>swath profile</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>420</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>551</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Baseline Layer</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Resolution orthogonal to baseline</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="cmdBrowseOutput">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="outputShapeBox"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>length of profile</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_6">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The directory where the output is stored.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>Output table</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="outputTableBox">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The directory where the output is stored.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>output profile lines shape</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QComboBox" name="inputBaselineBox"/>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QComboBox" name="inputRasterBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>Digital Terrain Model (raster)</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QDoubleSpinBox" name="resolutionBox">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="maximum">
       <double>99999.990000000005239</double>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="cmdBrowseOutputshp">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>resolution along baseline</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QDoubleSpinBox" name="distProfilesBox">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="maximum">
       <double>99999999.000000000000000</double>
      </property>
      <property name="value">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QDoubleSpinBox" name="lengthProfilesBox">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="maximum">
       <double>99999999.989999994635582</double>
      </property>
      <property name="value">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>inputRasterBox</tabstop>
  <tabstop>inputBaselineBox</tabstop>
  <tabstop>distProfilesBox</tabstop>
  <tabstop>lengthProfilesBox</tabstop>
  <tabstop>resolutionBox</tabstop>
  <tabstop>outputTableBox</tabstop>
  <tabstop>outputShapeBox</tabstop>
  <tabstop>cmdBrowseOutput</tabstop>
  <tabstop>cmdBrowseOutputshp</tabstop>
  <tabstop>button_box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>swathProfileDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>swathProfileDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
